front-end/playground/src/pages/Notifications/index.js

```diff
@@ -0,0 +1,36 @@
+import React, {useEffect, useState} from "react";
+import './style.css'
+import {getNotifications} from "../../services/User";
+
+const Notifications = (props) => {
+
+    const [notifications, setNotifications] = useState([])
+    useEffect(() => {
+
+        getNotifications().then((response) => {
```

Comment:
this may be wrapped by a setinterval method

Reply:
Thanks, I agree.